### ISSUE TITLE
[request] is it possible to invert mouse movement?

### ISSUE THREAD

**Issue by brjdk** [AlanChatham/FPSNav-for-SketchUp#1]
fps control is really needed in sketchup..... 
is it possible to invert mousemovement so it works like in fps games? 
when you move the mouse up - you look up, when you move the mouse down - you look down

thanks

**Comment by motionharvest** [comment]
Yes. Very small edit. [On line 343 of FPSNav.rb](https://github.com/AlanChatham/FPSNav-for-SketchUp/blob/master/FPSNav/FPSNav.rb#L343) invert _radiansToRotate_ like this:
`yRotationTransformation = Geom::Transformation.rotation(@FPSNav_eye, horizontalAxis, -radiansToRotate)`

**Comment by ThumperDone** [comment]
Hum, extensions in version 2018 are 'rbz'.  SO how would I install your 'rb' file?
Figured it out.. Copy the FPSNav folder to;  Sketchup2018 > ShippedExtensions. Then copy FPSNav.rb your folder to the root folder 'ShippedExtensions'.
